docker/README.md

```diff
@@ -0,0 +1,81 @@
+# Garage Docker
```

Comment:
I forgot to add a pre-requisites section, mainly for docker-ce and docker-compose.